packages/sdk-react/src/core/queries/useCounterpart.ts

```diff
@@ -441,6 +441,20 @@ export const useDeleteCounterpartContact = () => {
   );
 };
 
+export const useMakeCounterpartContactDefault = () => {
+  const { i18n } = useLingui();
+  const { api } = useMoniteContext();
+
+  return api.counterparts.postCounterpartsIdContactsIdMakeDefault.useMutation(
+    undefined,
+    {
+      onSuccess: () =>
+        toast.success(t(i18n)`Contact Person has been made default.`),
+      onError: (error) => toast.error(getAPIErrorMessage(i18n, error)),
```

Comment:
You can remove it because all errors are intercepted and displayed by `createQueryClient`.